src/core/Clipboard.cpp

```diff
@@ -35,6 +38,10 @@ void Clipboard::copy( JournallingObject * _obj )
 	QDomElement parent = doc.createElement( "Clipboard" );
 	_obj->saveState( doc, parent );
 	content[_obj->nodeName()] = parent.firstChild().toElement();
+
+	// Clear the QApplication clipboard, so we don't have any conflicts when LMMS has to
+	// decide between the QApplication clipboard and the internal clipboard data
+	QApplication::clipboard()->clear( QClipboard::Clipboard );
```

Comment:
This feels like an unnecessary/dangerous change. Have you checked all uses of the QApplication clipboard to ensure that this doesn't change copy/paste behavior at all?

Reply:
I didn't check every use of `QApplication::clipboard()`, but I'm not sure what's the danger in it. I mean, the clipboard is volatile, when you copy you add data to it but when you paste you can't trust the data will still be there (i.e.: if you copy something else before pasting). Maybe I'm missing something?

I had to do that because currently LMMS mixes the `Clipboard` class and the system's clipboard on copy/paste operations. The individual TCO copy uses `class Clipboard` but the multiple TCO copy uses the system's clipboard. So it's possible I'll have data on both simultaneously. When pasting I wouldn't be able to know which one should be pasted, so `Clipboard::copy` now clears the system's clipboard as well. Then I can know I should use it because the system's clipboard will be empty. Ideal would be if I cleared the `Clipboard` data as well when I copied multiple TCOs (which I don't currently do).

Another reason I advocate for changing everything to the system's clipboard, things can get confusing mixing both but I tried to come up with the cleanest solution possible, at least until the change happens (if everyone agrees). From a quick search it seems `Clipboard::copy` is only used on `src/core/Track.cpp` anyways, but other files uses `Clipboard::mimeType()` too.